ta/sims_keepalive/ta_sims_keepalive.c

```diff
@@ -0,0 +1 @@
+../sims/ta_sims.c
```

Comment:
I'm not sure we're very found of such file link.

A would prefer implementing back `sims_get_counter()` and `sims_entry_panic()` in the new TA.